src/config/containers-and-vms/libvirt.md

```diff
@@ -0,0 +1,28 @@
+# libvirt
+
+[libvirt](https://en.wikipedia.org/wiki/Libvirt) is an API and daemon for
```

Comment:
Do they have a homepage? Perhaps makes more sense to link it here.

Reply:
They do - https://libvirt.org/  - but i felt it was not particularly helpful for an introduction, at least not compared to the Wikipedia page. But yeah, you're right, it's probably more appropriate.

Reply:
I guess linking to the homepage is better for consistency sake? If you want to prioritize the more helpful page, I don't have any issue with it. Just wanted to bring it up.